crates/aptos/src/governance/mod.rs

```diff
@@ -352,17 +417,28 @@ impl CliCommand<TransactionSummary> for ExecuteProposal {
     }
 
     async fn execute(mut self) -> CliTypedResult<TransactionSummary> {
-        if !self.path.exists() {
-            return Err(CliError::UnableToReadFile(
-                self.path.display().to_string(),
-                "Path doesn't exist".to_string(),
-            ));
+        // TODO: Make one object so this code isn't copied
+        // Check script file
+        let script_path = self.script_path.as_path();
```

Comment:
All logic here and below seems redundant with same logic in create proposal. Can we refactor them into a separate func?